rundird.zig

```diff
@@ -54,6 +55,9 @@ pub fn main() !void {
     // while maintaining write permission to the root owned parent directory.
     _ = try os.prctl(os.PR.SET_SECUREBITS, .{os.SECBIT_NO_SETUID_FIXUP});
 
+    log.info("creating {}\n", .{build_options.rundir_parent});
+    os.mkdir(build_options.rundir_parent, 0o755) catch |err| if (err != error.PathAlreadyExists) return err;
```

Comment:
this should probably use the `makePath()` function from the std in case multiple levels of directories must be created.

Reply:
Oh nice, I searched for something like this but figured that since I didn't make it configurable and `/run` must exist it wasn't important.

Now it uses `makePath()`  and rundir-parent is a build option.